refinery/file_store/migrations/0002_filetypes_and_fileextensions.py

```diff
@@ -3,316 +3,11 @@
-        FileExtension(
-            name="fastqsanger",
-            filetype=Filetype.objects.get(name="FASTQSANGER")
-        ),
-        FileExtension(
-            name="fastqsolexa",
-            filetype=Filetype.objects.get(name="FASTQSOLEXA")
-        ),
-        FileExtension(
-            name="pdf",
-            filetype=Filetype.objects.get(name="PDF")
-        ),
-        FileExtension(
-            name="sam",
-            filetype=Filetype.objects.get(name="SAM")
-        ),
-        FileExtension(
-            name="seg",
-            filetype=Filetype.objects.get(name="SEG")
-        ),
-        FileExtension(
-            name="tabular",
-            filetype=Filetype.objects.get(name="TABULAR")
-        ),
-        FileExtension(
-            name="tdf",
-            filetype=Filetype.objects.get(name="TDF")
-        ),
-        FileExtension(
-            name="tgz",
-            filetype=Filetype.objects.get(name="TGZ")
-        ),
-        FileExtension(
-            name="txt",
-            filetype=Filetype.objects.get(name="TXT")
-        ),
-        FileExtension(
-            name="vcf",
-            filetype=Filetype.objects.get(name="VCF")
-        ),
-        FileExtension(
-            name="wig",
-            filetype=Filetype.objects.get(name="WIG")
-        ),
-        FileExtension(
-            name="xml",
-            filetype=Filetype.objects.get(name="XML")
-        ),
-        FileExtension(
-            name="bb",
-            filetype=Filetype.objects.get(name="BIGBED")
-        ),
-        FileExtension(
-            name="igv.tdf",
-            filetype=Filetype.objects.get(name="TDF")
-        ),
-        FileExtension(
-            name="zip",
-            filetype=Filetype.objects.get(name="ZIP")
-        ),
-        FileExtension(
-            name="fastq.gz",
-            filetype=Filetype.objects.get(name="FASTQ.GZ")
-        ),
-        FileExtension(
-            name="gct",
-            filetype=Filetype.objects.get(name="GCT")
-        ),
-        FileExtension(
-            name="fq",
-            filetype=Filetype.objects.get(name="FASTQ")
-        ),
-        FileExtension(
-            name="bcl",
-            filetype=Filetype.objects.get(name="BCL")
-        ),
-        FileExtension(
-            name="bcl.gz",
-            filetype=Filetype.objects.get(name="BCL GZip file")
-        ),
-        FileExtension(
-            name="bcl.tgz",
-            filetype=Filetype.objects.get(name="BCL TGZ file")
-        ),
-        FileExtension(
-            name="bai",
-            filetype=Filetype.objects.get(name="BAI file")
-        )
-    ]
-
-    for file_extension in file_extensions:
-        # Attempt to save FileExtension objects. If an IntegrityError is raised
-        # upon saving an object we continue as it already exists.
-        try:
-            with transaction.atomic():
-                file_extension.save()
-        except IntegrityError:
-            continue
+    pass
```

Comment:
Since this is an empty migration now, I'd simply remove [unnecessary code](https://docs.djangoproject.com/en/1.7/topics/migrations/#data-migrations)